build/tasks/debug.js

```diff
@@ -61,7 +61,7 @@ module.exports = function (grunt) {
         if (func) {
           // FunctionExpression.BlockStatement.body(Array)
           var funcBody = func.body.body;
-          esprima.parse('blocks.debug && blocks.debug.checkArgs(' + toValueString(data) + ', Array.prototype.slice.call(arguments), {})').body.forEach(function (chunk) {
+          esprima.parse('blocks.debug && blocks.debug.checkArgs(__DEBUG_METHOD, Array.prototype.slice.call(arguments), {}); var __DEBUG_METHOD = ' + toValueString(data) + ';').body.forEach(function (chunk) {
```

Comment:
escodegen generates code like that from it.
I'm not sure why but I can look into that this evening.
[Here](https://gist.github.com/Kanaye/7835cd4f0bfe67d1de13da43f88970e0#file-blocks-290b8ac6e758b9010b60ee2973fd582f51d8e23a-js-L59-L103) is a build of this pr showing that the code is generated like you mentioned above.